app/models/job_template_importer.rb

```diff
@@ -0,0 +1,26 @@
+class JobTemplateImporter
+  def self.import!(name, text, metadata)
+    return {
+      :status => false,
+      :result => 'Skipping job template import, remote execution plugin is not installed.'
+    } unless defined?(JobTemplate)
+    template = JobTemplate.import(
+      text.sub(/^name: .*$/, "name: #{name}").sub(/^model: .*$/, ''),
+      :update => true
+    )
+
+    c_or_u = template.new_record? ? 'Created' : 'Updated'
+    id_string = ('id' + template.id) rescue ''
+
+    if template.template != template.template_was
+      diff = Diffy::Diff.new(
```

Comment:
Ok, refactored this, in combination with https://github.com/theforeman/foreman_templates/commit/9efafedf039173a18d1d62c264d2b42902d8c398 it should work cleanly now